ppr-ui/src/components/unitNotes/UnitNoteReview.vue

```diff
@@ -1,23 +1,322 @@
+                      Mailing Address
+                    </th>
+                    <th class="px-0">
+                      Email Address
+                    </th>
+                    <th class="px-0">
+                      Phone Number
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td class="person-name">
+                      <v-icon class="mt-n2">
+                        {{ getMhrUnitNote.businessName ? 'mdi-domain' : 'mdi-account' }}
+                      </v-icon>
+                      <span class="font-weight-bold">
+                        {{ givingNoticeParty.personName.first }}
+                        {{ givingNoticeParty.personName.middle }}
+                        {{ givingNoticeParty.personName.last }}
+                      </span>
+                    </td>
+                    <td class="">
+                      <base-address
+                        :editing="false"
+                        :schema="PartyAddressSchema"
+                        :value="givingNoticeParty.address"
+                      />
+                    </td>
+                    <td class="">
+                      {{ givingNoticeParty.emailAddress }}
+                    </td>
+                    <td class="">
+                      {{ toDisplayPhone(givingNoticeParty.phoneNumber) }}
+                      <span v-if="givingNoticeParty.phoneExtension">
+                        Ext {{ givingNoticeParty.phoneExtension }}
+                      </span>
+                    </td>
+                  </tr>
+                </tbody>
+              </template>
+            </v-simple-table>
+          </div>
+        </section>
+      </div>
+    </v-card>
+
+    <section id="mhr-unit-note-person-giving-notice" class="mt-15">
+      <ContactInformation
+        :contactInfo="unitNoteSubmittingParty"
+        :content="{
+          title: '1. Submitting Party for this Change',
+          description: 'Provide the name and contact information for the person or business submitting ' +
+          ' this registration. You can add the submitting party information manually, or, if the submitting ' +
+          'party has a Personal Property Registry party code, you can look up the party code or name.',
+          sideLabel: 'Add Submitting Party'
+        }"
+        :validate="validate"
+        :setStoreProperty="setSubmittingParty"
+        @isValid="handleComponentValid(MhrCompVal.SUBMITTING_PARTY_VALID, $event)"
+      />
+    </section>
+
+    <section class="mt-15">
+      <h2>2. Effective Date and Time</h2>
+      <p class="mt-2">
+        Select the effective date and time for this {{ unitNoteType.header }}. Custom date and time can
+        be a date and time in the past. Notice of Caution will expire 90 days after the effective date.
+      </p>
+      // Placeholder for the component
+    </section>
+
+    <section class="mt-15">
```

Comment:
I would prefer not to have any margins in the components themselves. Because of the inconsistencies that you mention, we should let the parent to define the margins for each component where it's used.

Reply:
Also, I'm not using the Folio component.

Reply:
I am aware, that was just to demonstrate the margin differences, its the same for attention in my flow.